data/human/pirates/pirate intro.txt

```diff
@@ -0,0 +1,753 @@
+	choice
+		`	(This is my life now. Piracy, smuggling, the existence of the outlaw. I'm ready to embrace it.)`
+	`	Resolve will carry you far in this world. Tonight it's carried you to a spaceport surplus supplier; you return to your vessel with a pallet loaded with paint. Anoint your vessel in bellicose black and red, make your intentions known to the galaxy. There can be no pretense of peace when you fly girded in blood and midnight, but maybe that's exactly the way you want it to be.`
+	`	Some paint covers all manner of sin. Some speaks of the intention to commit yet more. You've just applied the latter sort to your ship.`
+
+
+
+mission "Pirate Overview"
+	landing
+	to offer
+		has "I Want To Be A Pirate: active"
+	on offer
+		conversation
+			`You think back to your conversation in that cramped, stinking drinking hole, of the colorful characters met and the tasks assigned:`
+			`	There's a worthy target - sure to attract the attention of every pirate with eyes and ears in the right places - cruising through Umbral soon. Ambush them, secure an antique lockbox, and then defend your prize against any scavengers.`
+			`	Alternatively, you can prove your worth through sheer spending power. Accrue one million credits and hand them off to someone referred to you by a stranger you just happened to meet in a bar, for the promise of words whispered into the right ears. You'll need to head to Smuggler's Den, an orbital installation in the Men system.`
+			`	Finally, you could murder a bunch of strangers and demand tribute from the survivors on pain of more ass-kicking. Specifically, strangers living on the planet of Darkstone. They probably deserve it anyways, or at least so says the cyborg who issued this task.`
+			`	Completing any of these truly important quests and reporting back to the ne'er-do-wells on Greenrock will make you a pirate.`
+				decline
+
+
+
+mission "Pirate Intro: Boarding"
+	landing
+	name "Board a merchant"
+	description `Board a merchant.`
+	destination "Greenrock"
+	waypoint "Umbral"
+	to offer
+		has "I Want To Be A Pirate: active"
+	to fail
+		has "I Want To Be A Pirate: done"
+	on offer
+		event "add merchant targets"
+	to complete
+		has "Plunder Defense: done"
+
+fleet "Merchant Targets"
+	government "Merchant (Target)"
+	names "civilian"
+	cargo 1
+	personality
+		heroic
+	variant 1
+		"Freighter"
+
+government "Merchant (Target)"
+	swizzle 5
+	
+	"player reputation" 10
+	"attitude toward"
+		"Pirate" -.2
+		"Korath" -.2
+	"bribe" .05
+	"fine" 0
+	"friendly hail" "friendly civilian"
+	"hostile hail" "hostile civilian"
+
+
+
+mission "Plunder Defense"
+	name "Return with spoils"
+	description `Defeat your rival pirates in Umbral, then return to <destination> with the spoils from the <origin> to confirm your loyalty.`
+	boarding
+	source
+		government "Merchant (Target)"
```

Comment:
Ditch the mission-specific government and instead have this occur on any merchant, but you may want to select a few specific systems in which this interaction can occur.

Reply:
Part of Umbral was to have an out of the way location where those merchants are up to something, then bringing the fight for the plunder between you and the pirates. The natural spawn rates are a bit too low for normal merchants, and other nearby uninhabited systems have much higher traffic from either merchants, militia/FW/Republic, or both. 

Could use a mission NPC, which would hover around too much _or_ be too hard to nail down, or add in the fleet definition anyways for those dead ends which is basically what's currently in use. I do like using a more elegant solution on paper in terms of a broader boarding mission, but I don't think there's a good fit with what's naturally in the game. The extra fleet event could stay with just the normal Merchant government, but I suspect future pirate jobs/missions could still make use of such a government, as well. If it has to be natural-spawning merchants, what system(s) should be used then?